### PR TITLE
Feature/napsspo 474

### PR DESCRIPTION
**Pull request by kharyam**
Implements the following acceptance criteria from [NAPSSPO-473](https://projects.engineering.redhat.com/browse/NAPSSPO-474)

1. Update container image names

- quay.io/tssc/tssc-base (unchanged)
- quay.io/tssc/tssc-ci-agent-jenkins (jnlp agent only)
- quay.io/tssc-tool-maven (maven only)

2. Refactor the jenkins agent image to only have the jnlp client
3. Fix the name labels in the Dockerfiles to use "tssc/..." instead of "openshift/..."
4. Parameterize the FROM statements so that they can be pointed to local images for testing purposes
5. Update the README with instructions for building locally

Note - The Badges in the README will automatically update properly once this is merged

### CODE REVIEW

File: .github/workflows/tssc-tool-maven-pr.yaml [hunk 1]
```diff
@@ -30,5 +30,4 @@ jobs:
           echo "Running: docker run ${image_name}:${{ steps.check_version.outputs.IMAGE_TAGS }} mvn --version"
           docker run -u 1001 ${image_name}:${{ steps.check_version.outputs.IMAGE_TAGS }} mvn --version
           echo "Running: docker run ${image_name}:${{ steps.check_version.outputs.IMAGE_TAGS }} -u 1001 touch ~/tssc/.m2/test.txt"
-          docker run -u 1001 ${image_name}:${{ steps.check_version.outputs.IMAGE_TAGS }} touch /home/tssc/.m2/test.txt 
-         
+          docker run -u 1001 ${image_name}:${{ steps.check_version.outputs.IMAGE_TAGS }} touch /home/tssc/.m2/test.txt
```
Comment (HunterGerlach) [inline .github/workflows/tssc-tool-maven-pr.yaml:33]: Newline EOF needed

File: tssc-tool-maven/contrib/bin/entrypoint [hunk 2]
```diff
@@ -88,3 +88,5 @@ fi
 
 CONTAINER_MEMORY_IN_BYTES=$(cat /sys/fs/cgroup/memory/memory.limit_in_bytes)
 CONTAINER_MEMORY_IN_MB=$((CONTAINER_MEMORY_IN_BYTES/2**20))
+
+$@
```
Comment (HunterGerlach) [inline tssc-tool-maven/contrib/bin/entrypoint:92]: Newline EOF needed

File: .github/workflows/tssc-tool-maven-pr.yaml (hunk 1 above)
Comment (kharyam) [reply .github/workflows/tssc-tool-maven-pr.yaml]: Done

File: tssc-tool-maven/contrib/bin/entrypoint (hunk 2 above)
Comment (kharyam) [reply tssc-tool-maven/contrib/bin/entrypoint]: Done

**Pull request merged**